apps/server/src/shared/domain/entity/task-card.entity.ts

```diff
@@ -39,6 +42,10 @@ export class TaskCard extends BaseEntityWithTimestamps implements ICard, ITaskCa
 	@ManyToOne('User', { fieldName: 'userId' })
 	creator!: User;
 
+	@Index()
+	@ManyToOne('Course', { fieldName: 'courseId', nullable: true })
```

Comment:
ok, if we let it be null, then in some comments above, it could be null instead of undefined

Reply:
you are right, removed the nullable to be consistent as undefined without the possibility to be `null`

Reply:
this one seems to be needed, as otherwise it does not allow the value to be undefined somehow, even if its a separate check, so nullable seems to cover more then just `null`

Reply:
yeah, comes from relational databases, mikrorm mixes up the terms...